### PR TITLE
Add Phpredis

### PR DESCRIPTION
**Pull request by hacfi**
https://github.com/phpredis/phpredis

If this doesn’t make 1.0 I’ll remove the `@since 1.0` PhpDoc.

### CODE REVIEW

**Comment by hacfi** [comment]
I also added `redis-server` for Travis CI..phpunit now runs both PhpRedis and Predis tests.

**Comment by webmozart** [comment]
Cool stuff! :)

File: src/PhpRedisStore.php [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+namespace Webmozart\KeyValueStore;
+
+use Exception;
+use Redis;
+use Webmozart\KeyValueStore\Api\KeyValueStore;
+use Webmozart\KeyValueStore\Api\SerializationFailedException;
+use Webmozart\KeyValueStore\Assert\Assert;
+
+/**
+ * A key-value store that uses PhpRedis to connect to a Redis instance.
```
Comment (webmozart) [inline src/PhpRedisStore.php]: "...uses the PhpRedis extension to..."

File: src/PhpRedisStore.php [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+ *
+ * @since  1.0
+ * @author Bernhard Schussek <bschussek@gmail.com>
+ * @author Philipp Wahala <philipp.wahala@gmail.com>
```
Comment (webmozart) [inline src/PhpRedisStore.php]: @link https://github.com/phpredis/phpredis

**Comment by webmozart** [comment]
Apart from my comments, this looks good! Thanks! :)

**Comment by hacfi** [comment]
Done.

After it’s merged I’ll try to get memcache, memcached and riak running on Travis CI.

**Pull request merged**